lib/wda/webdriveragent.js

```diff
@@ -155,10 +161,23 @@ class WebDriverAgent {
     }
   }
 
+  /**
+   * Uninstall WDAs from the test device.
+   * Over Xcode 11, multiple WDA can be in the device since Xcode 11 generates different WDA.
+   * Appium does not expect multiple WDAs are running on a device.
+   */
   async uninstall () {
-    log.debug(`Removing WDA application from device`);
     try {
-      await this.device.removeApp(WDA_BUNDLE_ID);
+      const bundleIds = await this.getInstalledWDBundleIds();
+      if (_.isEmpty(bundleIds)) {
+        log.debug('No WDAs on the device.');
+        return;
+      }
+
+      log.debug(`Uninstalling WDAs: '${bundleIds}'`);
+      await B.each(bundleIds, async (bundleId) => {
```

Comment:
and here I would revert back to `for of`. `each` call does not provide any preferences over a normal loop